test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +240,23 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should run virtualbox installer executable to extract msi installer into temp folder', function(done) {
+      let stub = sandbox.stub(child_process, 'execFile').yields(null, "Files were extracted to temp", "");
+      try {
+        installer.install(fakeProgress, success, failure);
+        done();
```

Comment:
calling done inside main thread is wrong, it should be inside callback, please remove it here and form it second parameter callback.